src/governance.cpp

```diff
@@ -947,8 +948,8 @@ bool CGovernanceManager::ProcessVote(CNode* pfrom, const CGovernanceVote& vote,
         std::ostringstream ostr;
         ostr << "CGovernanceManager::ProcessVote -- Old invalid vote "
                 << ", MN outpoint = " << vote.GetVinMasternode().prevout.ToStringShort()
-                << ", governance object hash = " << vote.GetParentHash().ToString() << "\n";
-        LogPrintf(ostr.str().c_str());
+                << ", governance object hash = " << vote.GetParentHash().ToString();
+        LogPrintf("%s\n", ostr.str());
```

Comment:
I fixed only few places where I noticed it's happening but I see many more places like that... Should probably be fixed too for consistency. Thoughts?